fmslack/cli.py

```diff
@@ -85,9 +85,9 @@ def slack(redis_uri, redis_channel, slack_webhook_url, api_url, log_level):
                 logger.info('Event: PLAY')
                 track = query_api(api_url, data['uri'])
                 if track is not None:
-                    logger.debug(
-                        'API returned track data for {0}'.format(track['spotify_uri']))
-                    slack = slack_post(
```

Comment:
``slack`` was defined but never used, its generally bad practice to define variables and not use them :smile: